### PR TITLE
Add transfer strings

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -68,11 +68,19 @@
             },
             "notSupportedMedia": "서프는 곧 이 True NFT를\n표시하게 될 것입니다.",
             "owner": "소유자",
+            "request": {
+                "title": "Request"
```
Comment (sertony) [inline src/ko.json]: @kkkhesha can you translate?

File: src/ko.json [hunk 2]
```diff
@@ -68,11 +68,19 @@
...
+            },
             "sections": {
                 "library": "라이브러리",
                 "partners": "파트너",
                 "trending": "유행"
             },
+            "send": {
+                "confirm": "Confirm",
+                "receiver": "Receiver",
+                "title": "Send"
```
Comment (sertony) [inline src/ko.json]: Same

File: src/ru.json [hunk 3]
```diff
@@ -67,11 +67,19 @@
                 "notTrue": "Not True",
                 "trueNft": "True NFT"
             },
+            "request": {
+                "title": "Request"
```
Comment (sertony) [inline src/ru.json]: @AnnStepanova can you translate?

File: src/ru.json [hunk 4]
```diff
@@ -67,11 +67,19 @@
...
+            },
             "sections": {
                 "library": "Библиотека",
                 "partners": "Партнеры",
                 "trending": "Тренды"
             },
+            "send": {
+                "confirm": "Confirm",
+                "receiver": "Receiver",
+                "title": "Send"
```
Comment (sertony) [inline src/ru.json]: Same

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [reply src/ko.json]:
```suggestion
                "title": "요청"
```

File: src/ko.json [hunk 5]
```diff
@@ -68,11 +68,19 @@
...
+            "request": {
+                "title": "Request"
+            },
             "sections": {
                 "library": "라이브러리",
                 "partners": "파트너",
                 "trending": "유행"
             },
+            "send": {
+                "confirm": "Confirm",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "confirm": "확인",
```

File: src/ko.json [hunk 6]
```diff
@@ -68,11 +68,19 @@
...
+                "receiver": "Receiver",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "receiver": "수신인",
```

File: src/ko.json (hunk 2 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "title": "보내기"
```

File: src/ru.json (hunk 3 above)
Comment (AnnStepanova) [reply src/ru.json]: "title": "Запрос"

File: src/ru.json (hunk 4 above)
Comment (AnnStepanova) [reply src/ru.json]: "confirm": "Подтвердить",
                "receiver": "Кому",
                "title": "Отправка"

**Review by sertony — APPROVED** [review]

**Pull request merged**